zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ClusterEndpointIT.java

```diff
@@ -75,6 +75,32 @@ void shouldRequestPartitionLeave() {
     }
   }
 
+  @Test
+  void shouldRequestClusterPurge() {
+    final ClusterActuator actuator;
+    try (final var cluster = createCluster(1)) {
```

Comment:
:wrench: Let's increase the replication factor - it's a cheap way to get more test coverage imo.

Reply:
Actually I can't get this test to run properly in this PR, because of the leave-partition-failcheck, which is changed in a coming PR. So I rewrote this test to expect this exception, and I will provide the real test in the coming PR. So in this version we can at least ensure, that the endpoint is callable and not throwing random exceptions.

Reply:
Then let's add a TODO and come back to it later.